backend/models/time.py

```diff
@@ -161,10 +166,11 @@ def closest_with_early_exit(self, origin, point_list, exit_distance):
         ]
 
         if not relevant_points:
-            logger.warning(
-                'No relevant points, returning an absurdly large time for {}'.format(origin)
+            logger.warning('No relevant points, returning a very large number * min_haversine.')
+            return (
+                min_haversine_distance / TIME_TO_DISTANCE_HOURS_TO_METERS * 60.0 * 2,
```

Comment:
is this still a very large number?

Reply:
Perhaps `returning min_haversine_distance * fudge_factor`? 😂